docs/config/README.md

```diff
@@ -409,6 +409,31 @@ The module will be shown if any of the following conditions are met:
 symbol = "⚙️ "
 ```
 
+## Time
+
+The `time` module shows the current **local** time.
+
+::: tip
+For the `format` configuration value, take a look [here](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to see what you can pass in.
+:::
+
+### Options
+
+| Variable   | Default | Description                                  |
+| ---------- | ------- | -------------------------------------------- |
+| `12hr`     | `false` | Enables 12 hour formatting                   |
+| `format`   | `''`    | Enables the user to set a custom time format |
+| `disabled` | `false` | Disables the `username` module.              |
```

Comment:
Should this say "Disables the `time` module."?

```suggestion
| `disabled` | `false` | Disables the `time` module.              |
```

Reply:
It totally should @LinusU ... thanks for reviewing!